### DISCUSSION TITLE
Copier includes jinja2-ansible-filters, which appears to be GPL3. Does this poison the MIT nature of the project?

### DISCUSSION THREAD

**Discussion by alan-copeland-keysight** [copier-org/copier#1397] [Q&A]
Hi, doing a license check as part of a project and ran across this conundrum. jinja2-ansible-filters appears to be GPL3. Wouldn't that poison the MIT license of this project? I do see that "some" parts of jinja2-ansible-filters appear to be BSD, but are those the "only" parts you are using?

**Comment by yajo** [comment]
It seems like the license of jinja2-ansible-filters is [a mix between GPL3 and BSD](https://gitlab.com/dreamer-labs/libraries/jinja2-ansible-filters/-/blob/58e352012f1b0bd6eb68cd02a4618f9af21d61bb/LICENSE). Weird, because that's essentially illegal AFAIK. They copied GPL3 code from ansible and wrote BSD code, which should be also infected by the GPL one, so we can consider that project as completely GPL3.

The rest is a duplicate of #1398. Let's continue there.
